tests/lib/hardware/libvirt.py

```diff
@@ -57,12 +59,15 @@ def __init__(self, name, role, tags, conn, image_path,
         self._network = network
         self._disk_number = disk_number,
         self._memory = memory * 1024 * 1024
+        self._workspace = workspace
         self._ssh_public_key = workspace.public_key
         self._ssh_private_key = workspace.private_key
         self._snap_img_path = os.path.join(
             workspace.working_dir, f"{self.name}-snapshot.qcow2")
         self._cloud_init_seed_path = os.path.join(
             workspace.working_dir, f"{self.name}-cloud-init-seed.img")
+        self._disk_template = os.path.abspath(os.path.join(
```

Comment:
not needed if we just use the inline template

Reply:
As I'm not in favor of inline strings this would still be necessary.

Reply:
Done